packages/e2e-tests/pages/patients/AllPatientsPage.ts

```
@@ -1,28 +1,329 @@
+  readonly NewPatientMaleChk: Locator;
+  readonly NewPatientFemaleChk: Locator;
+  readonly NewPatientNHN: Locator;
+  readonly NewPatientConfirmBtn: Locator;
+  _patientData?: {
+    id: string;
+    firstName: string;
+    lastName: string;
+    gender: string;
+    formattedDOB: string;
+    nhn: string;
+    village: String,
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)